Libraries/libcar/Tests/test_Writer.cpp

```diff
@@ -63,9 +63,24 @@ static test_car_key_format keyfmt_s = {
 
 static struct car_key_format *keyfmt = &keyfmt_s.keyfmt;
 
-TEST(Writer, TestWriter)
+class WriterTest : public testing::Test {
+
+protected:
+    std::string tmpfilename;
+
+    virtual void SetUp() {
+        std::ostringstream stream;
+        stream << "/tmp/test_car_Writer_output." <<  ::getpid() << ".car";
```

Comment:
I guess it could, but this works pretty well, and it does also test the file backed mmap path which had bugs.